.github/workflows/deploy.yml

```diff
@@ -23,9 +23,12 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
           VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
-          VERCEL_PROJECT_WALLETS: ${{ secrets.VERCEL_PROJECT_WALLETS }}
```

Comment:
In my opinion, if we don't need these packages
Check in the **_listPackagesToBeDeployed_** function
If the project ID does not exist, it will not be included in the output value

Reply:
Good catch! Thanks.
I fixed it in my last commit. All the clients can be deployed again as before.